presto-benchmark-driver/src/main/java/io/prestosql/benchmark/driver/BenchmarkQueryRunner.java

```diff
@@ -122,13 +123,13 @@ public BenchmarkQueryResult execute(Suite suite, ClientSession session, Benchmar
         double[] queryCpuTimeNanos = new double[runs];
         for (int i = 0; i < runs; ) {
             try {
-                long startCpuTime = getTotalCpuTime();
+                long startCpuTime = getTotalCpuTime(user);
```

Comment:
wondering about using `session.getUser()` here as opposed to a new member variable `user`. 

I think having this new member variable `user` makes sense, since getting cpu time is like a monitoring request, as opposed to a benchmark query fired by the session user. 

In future, we can probably provide a different pre-configured "super-user"  in this constructor invocation from `BenchmarkDriver`, instead of `clientSession.getUser`.

Reply:
`BenchmarkQueryRunner` does not have the `clientSession` inside. Do you mean we can pass the `clientSession` to `BenchmarkQueryRunner` instead of passing server and user configuration respectively?

Reply:
> wondering about using session.getUser() here as opposed to a new member variable user.

^ for this, I just meant getting `session.getUser` in the `execute` method.

> In future, we can probably provide a different pre-configured "super-user" in this constructor invocation from BenchmarkDriver, instead of clientSession.getUser.

^ for this, I meant passing in a different user in BenchmarkQueryRunner constructor.